alvr/server/src/connection.rs

```diff
@@ -727,6 +729,7 @@ fn connection_pipeline(
                 } else {
                     continue;
                 };
+                #[cfg(windows)]
```

Comment:
Either make this not linux, so it's good for both macos and windows or put it back into a big windows block

Reply:
i think i will put it into big windows block...

Reply:
This removed macos audio, should be reverted